widget.py

```diff
@@ -304,6 +307,12 @@ def updateActState(self, message):
                 elif(message.state == packet_spec.ActuatorState.ON):
                     self.ui.igniterState.setText("OPEN")
 
+    #Creates a file or overwrites existing one, and writes the text in the logOutput into the file
+    def save_to_file(self):
+        f = open("log_data.txt", "w")
+        f.write(self.ui.logOutput.toPlainText())
+        f.close();
+
```

Comment:
Can we store these in a directory named `logs`. Also could you have the name of the file be the current time and data?